src/components/UI/Display/MAPI/CLIGuide/CLIGuideAdditionalInfo.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Heading, Text } from 'grommet';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledLink = styled.a`
+  color: ${({ theme }) => theme.global.colors['input-highlight']};
+`;
+
+interface ICLIGuideAdditionalInfoLink {
+  label: string;
+  href: string;
+  external?: boolean;
+}
+
+interface ICLIGuideAdditionalInfoProps
+  extends React.ComponentPropsWithoutRef<'div'> {
+  links?: ICLIGuideAdditionalInfoLink[];
+}
+
+const CLIGuideAdditionalInfo: React.FC<ICLIGuideAdditionalInfoProps> = ({
+  links,
+  ...props
+}) => {
+  return (
+    <Box {...props}>
+      <Box>
+        <Heading level={5}>Additional information</Heading>
+      </Box>
+      <Box>
+        {links?.map((link) => (
+          // eslint-disable-next-line react/jsx-no-target-blank
```

Comment:
This is a false positive, we provide `noopener noreferrer` when `_blank` is used.